common/logger.go

```diff
@@ -15,8 +16,24 @@ type Logger struct {
 	LogFile *os.File
 }
 
-func CreateLogger(logFile string) *Logger {
-	f, err := os.OpenFile(logFile, os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0644)
+func CreateLogger(logPath string, logFile string, levels ...slog.Level) *Logger {
+	var level slog.Level
+	if len(levels) > 0 {
+		level = levels[0]
+	} else {
+		level = slog.LevelInfo
+	}
```

Comment:
```suggestion
	level := slog.LevelInfo
	if len(levels) > 0 {
		level = levels[0]
	}
```

assigning default first can be more readable and make for compact and smaller code